src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/NetworkWatcherAPITests.ps1

```diff
@@ -499,7 +474,7 @@ function Test-Troubleshoot
     # Setup
     $resourceGroupName = Get-ResourceGroupName
     $nwName = Get-ResourceName
-    $location = "centraluseuap"
+    $location = "westcentralus"
```

Comment:
@irrogozh are we able to apply the same `Get-Location` function to this tests as well?

Reply:
done